TheBindingOfIssac/TheBindingOfIssac/View/SideMenuView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SideMenuView.swift
+//  TheBindingOfIssac
+//
+//  Created by Marcin Dytko on 06/12/2022.
+//
+
+import SwiftUI
+
+struct SideMenuView: View {
+    // ToDo: Change this viewModel
+    @ObservedObject var viewModel: HomeViewModel
+    
+    var body: some View {
+        
+            VStack{
+                HStack{
+                    Text("Game assistant")
+                        .foregroundColor(.white)
+                        .font(.headline)
+                    Image("Bum-bo_the_Brave")
```

Comment:
This is not a huge deal but if you are using the same code:
```
HStack{
  Text()
  Image("")
}
```
more than once it's a good idea to create a custom struct/class from it to not repeat the code. Take it as a hint not something that is crucial for the app development.